scripts/windows_install_perl.ps1

```diff
@@ -1,6 +1,6 @@
 $ErrorActionPreference = "Stop"
 
-$perl_version = $args[0]
+$perl_version = $Env:TEMP_PERL_VERSION
```

Comment:
These changes mean that the dockerfiles don't work anymore, correct? Yet they're not removed...

Reply:
Yes, they don't work anymore.

Reply:
Then the relevant files should be removed...